src/helpers/http.ts

```diff
@@ -0,0 +1,79 @@
+import { IHttp, IHttpRequest, IHttpResponse } from '@rocket.chat/apps-engine/definition/accessors';
+import { IUser } from '@rocket.chat/apps-engine/definition/users';
+
+import { TodoistApp } from '../../TodoistApp';
+
+export type HttpMethod = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH';
+
+export interface IHttpOptions extends IHttpRequest {
```

Comment:
I think we can remove this class in favor of service classes that wrap the IHttp from apps engine